app/routes/_public.terms.tsx

```diff
@@ -0,0 +1,16 @@
+import { PolicyContainer } from '~/components/PolicyContainer';
+
+const TERMS_CONTENT = `
```

Comment:
would it be possible to define the content in the actual md file and have this component read it from there? would make it easier to update the content.

also I think we should use [static pre-rendering](https://reactrouter.com/start/framework/rendering#static-pre-rendering) for static pages like this